### PR TITLE
add cart icon badge with cart items count

### LINKED ISSUE #239: Show quantity of products in cart as an cart icon badge

### PR DESCRIPTION
**Pull request by Qrzy**
- [x] I Accept [Individual Contributor License Agreement](https://github.com/DivanteLtd/next/blob/master/CLA.md)

Closes #239

### CODE REVIEW

File: packages/core/theme-module/theme/components/AppHeader.vue [hunk 1]
```diff
@@ -3,40 +3,49 @@
...
 import uiState from '~/assets/ui-state';
+import { useCart } from '<%= options.composables %>';
+import { getCartTotalItems } from '<%= options.helpers %>';
+import { computed } from '@vue/composition-api';
 const { toggleCartSidebar, toggleLoginModal } = uiState;
 
 export default {
   setup() {
+    const { cart } = useCart();
+    const cartTotalItems = computed(() => {
```
Comment (filrak) [inline packages/core/theme-module/theme/components/AppHeader.vue:52]: @krskibin domerguj ;d

File: packages/core/theme-module/theme/components/AppHeader.vue [hunk 2]
```diff
@@ -3,40 +3,49 @@
...
+      return getCartTotalItems(cart.value).toString();
```
Comment (filrak) [inline packages/core/theme-module/theme/components/AppHeader.vue:53]: Why this toString is needed?

**Review by filrak — APPROVED** [review]

File: packages/core/theme-module/theme/components/AppHeader.vue (hunk 1 above)
Comment (Qrzy) [reply packages/core/theme-module/theme/components/AppHeader.vue:52]: what is this about?

File: packages/core/theme-module/theme/components/AppHeader.vue (hunk 2 above)
Comment (Qrzy) [reply packages/core/theme-module/theme/components/AppHeader.vue:53]: The `cartItemsQty` prop on `SfHeader` is declared as `String` so it pollutes the console with warnings when we put number there. 
Probably it should be reported to UI team, quantity is always a number...

**Review by patzick — APPROVED** [review]

**Review by andrzejewsky — APPROVED** [review]

**Pull request merged**